CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Documentation for rocPRIM is available at
 
 ### Changed
 
+* Tuned device segmented radix sort for gfx942.
+* Various optimizations to radix rank and radix sort.
```

Comment:
Because this is expanded on below, I would remove this line and leave the two below. It isn't needed.